### PR TITLE
Docs review

### PR DESCRIPTION
**Pull request by 0xGabi**
Note I did a first commit where I included lint to the markdown files. The actual documentation changes are on this commit: https://github.com/aragon/connect/pull/72/commits/3a32151a41b0acf3b18802c26dd19fd678178588

### CODE REVIEW

File: api-reference/connect.md [hunk 1]
```diff
@@ -6,20 +6,20 @@ This file documents the main exports of the library.
...
+| returns               | `Promise<Organization>`                   | An `Organization` instance.                                                                              |
 
 ### Errors
 
-| Type | Description |
-| :--- | :--- |
-| `ConnectionError` | Gets thrown if the connection fails. |
+| Type                   | Description                                            |
+| :--------------------- | :----------------------------------------------------- |
+| `ConnectionError`      | Gets thrown if the connection fails.                   |
```
Comment (0xGabi) [inline api-reference/connect.md:22]: We don't have support for Errors yet. Should we include a quick PR to include them or remove this from the documentation for now?

File: packages/connect-core/src/entities/Role.ts [hunk 2]
```diff
@@ -29,14 +29,16 @@ export default class Role extends CoreEntity implements RoleData {
...
-      const role = roles.find((role) => role.bytes === data.hash)
+      const role = roles.find(role => role.bytes === data.hash)
 
       this.name = role?.id
       this.description = role?.name
     }
 
     this.appAddress = data.appAddress
-    this.grantees = data.grantees
+    this.grantees = data.grantees?.map(
```
Comment (0xGabi) [inline packages/connect-core/src/entities/Role.ts:39]: I decided to create a `Permission` instance for every grantee entity.

File: api-reference/organization.md [hunk 3]
```diff
@@ -8,149 +8,149 @@ An `Organization` instance represents an Aragon organization and allows to inter
```
Comment (0xGabi) [inline api-reference/organization.md:8]: We still don't have support for several of the API methods on this document. Should we remove them from the docs now?

File: api-reference/role.md [hunk 4]
```diff
@@ -4,10 +4,12 @@ A single role, which can get assigned to create a permission.
...
-| Name      | Type       | Description                                                           |
-| :-------- | :--------- | :-------------------------------------------------------------------- |
-| `name`    | `String`   | Name of the role. E.g. `"Mint tokens"`.                               |
-| `id`      | `String`   | Identifier of the role. E.g. `"MINT_ROLE"`.                           |
-| `params`  | `String[]` | Params associated to the role. E.g. `[ "Receiver", "Token amount" ]`. |
-| `bytes`   | `String`   | Encoded identifier for the role.                                      |
-| `manager` | `String`   | Address of the role manager.                                          |
+| Name          | Type           | Description                                                           |
+| :------------ | :------------- | :-------------------------------------------------------------------- |
+| `appAddress`  | `String`       | App address.                                                          |
```
Comment (bpierre) [inline api-reference/role.md:9]:
```suggestion
| `appAddress`  | `String`       | Address of the app this role belongs to.                              |
```

File: api-reference/transaction-intent.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
-# Intent
+# Transaction Intent
```
Comment (bpierre) [inline api-reference/transaction-intent.md:1]:
```suggestion
# TransactionIntent
```

File: api-reference/transaction-intent.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
...
 
-An TransactionIntent to create an Ethereum transaction on, or through, an Aragon organization or its apps. It allows to retrieve transaction paths or a set of transactions to sign.
+An Transaction Intent to create an Ethereum transaction on, or through, an Aragon organization or its apps. It allows to retrieve transaction paths or a set of transactions to sign.
```
Comment (bpierre) [inline api-reference/transaction-intent.md:3]:
```suggestion
A transaction intent to create an Ethereum transaction on, or through, an Aragon organization or its apps. It allows to retrieve transaction paths or a set of transactions to sign.
```

File: packages/connect-core/src/entities/Role.ts (hunk 2 above)
Comment (bpierre) [reply packages/connect-core/src/entities/Role.ts:39]: Should we name it `permissions` then?

File: api-reference/connect.md (hunk 1 above)
Comment (bpierre) [reply api-reference/connect.md:22]: I think we could try to add proper errors (I am thinking in particular about `OrganizationNotFound` which is very common), and we could remove them just before releasing if we are not ready.

File: SUMMARY.md [hunk 7]
```diff
@@ -1,22 +1,22 @@
...
 
-* [⚡ Aragon Connect](README.md)
+- [⚡ Aragon Connect](README.md)
 
 ## GUIDES
 
-* [Getting started](guides/getting-started.md)
-* [Aragon app subgraph](guides/subgraph-template.md)
+- [Getting started](guides/getting-started.md)
+- [Aragon app subgraph](guides/subgraph-template.md)
```
Comment (bpierre) [inline SUMMARY.md:8]:
```suggestion
- [Creating an app subgraph](guides/subgraph-template.md)
```

File: SUMMARY.md [hunk 8]
```diff
@@ -1,22 +1,22 @@
...
+- [The Graph connectors](guides/thegraph-connectors.md)
```
Comment (bpierre) [inline SUMMARY.md:9]:
```suggestion
- [Using Connect with The Graph](guides/thegraph-connectors.md)
```

File: api-reference/connect.md (hunk 1 above)
Comment (0xGabi) [reply api-reference/connect.md]: I agree we should include at least a handler for `OrganizationNotFound `. But let's do it in a different PR.

File: api-reference/organization.md (hunk 3 above)
Comment (0xGabi) [reply api-reference/organization.md]: Is this still necessary with our current GitBook setup?

I will go ahead and give a stab to include the few API methods we miss. I believe we already have all the logic we need.

**Pull request merged**